libs/bixarena/api-description/src/components/schemas/LeaderboardSort.yaml

```diff
@@ -2,9 +2,6 @@ type: string
 description: The field to sort leaderboard entries by.
 enum:
   - rank
-  - bt_score
-  - vote_count
   - created_at
-  - model_name
 default: rank
-example: bt_score
+example: rank
```

Comment:
By default, the leaderboard will be sorted by rank (+ secondary sort  `btScore` processed in backend); also remove unused sorting.